docs/platforms/javascript/common/profiling/browser-profiling.mdx

```diff
@@ -0,0 +1,83 @@
+---
+title: Browser Profiling
+sidebar_order: 5000
+supported:
+  - javascript
+notSupported:
+  - javascript.bun
+  - javascript.cordova
+  - javascript.capacitor
+  - javascript.deno
+  - javascript.cloudflare
+  - javascript.node
+  - javascript.aws-lambda
+  - javascript.azure-functions
+  - javascript.connect
+  - javascript.express
+  - javascript.fastify
+  - javascript.gcp-functions
+  - javascript.hapi
+  - javascript.koa
+  - javascript.nestjs
+description: "Collect & view performance insights for JavaScript programs with Sentry’s JavaScript Profiling integration. Get started with browser profiling here."
+---
+
+<Note>
+
+Browser Profiling is currently in beta. Beta features are still in progress and may have bugs. We recognize the irony.
+
+</Note>
+
+The browser profiling integration is built using [JS Self-Profiling API](https://wicg.github.io/js-self-profiling/) and will likely only move out of beta once the specification progresses and gains adoption. See platform [status](https://chromestatus.com/feature/5170190448852992).
+
+Note that since the profiling API is currently only exposed in Chromium, profiles collected only include that demographic. This is obvious, but should not be forgotten when analyzing the data collected. We hope that as the API gains adoption, other browsers will implement it as well.
```

Comment:
```suggestion
Note that since the profiling API is currently only exposed in Chromium, profiles collected only include that demographic. We hope that as the API gains adoption, other browsers will implement it as well.
```